samples/bannerSample/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.annotation:annotation:1.0.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'com.inmobi.monetization:inmobi-ads:9.1.1'
+    implementation 'com.inmobi.monetization:inmobi-ads:9.1.6'
 
     implementation 'com.google.android.gms:play-services-ads-identifier:17.0.0'
     implementation 'com.google.android.gms:play-services-location:16.0.0'
```

Comment:
Let's use project-level Gradle to maintain dependency versions, also match the versions with supported deps versions